specification/security/resource-manager/Microsoft.Security/preview/2021-07-01-preview/customAssessmentAutomation.json

```diff
@@ -0,0 +1,335 @@
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/customAssessmentAutomationName"
+          },
+          {
+            "$ref": "#/parameters/customAssessmentAutomationBody"
+          }
+        ],
+        "x-ms-examples": {
+          "Create a Custom Assessment Automation": {
+            "$ref": "./examples/CustomAssessmentAutomations/customAssessmentAutomationCreate_example.json"
+          }
+        },
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/customAssessmentAutomation"
+            }
+          },
+          "200": {
+            "description": "OK - Updated",
+            "schema": {
+              "$ref": "#/definitions/customAssessmentAutomation"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "summary": "Deletes a custom assessment automation",
+        "description": "Deletes a custom assessment automation by name for a provided subscription",
+        "tags": [
+          "customAssessmentAutomations"
+        ],
+        "operationId": "CustomAssessmentAutomations_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/customAssessmentAutomationName"
+          }
+        ],
+        "x-ms-examples": {
+          "Delete a Custom Assessment Automation": {
+            "$ref": "./examples/CustomAssessmentAutomations/customAssessmentAutomationDelete_example.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK - Custom Assessment Automation was deleted"
+          },
+          "204": {
+            "description": "No Content - Custom Assessment Automation does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Security/customAssessmentAutomations": {
+      "get": {
+        "summary": "List custom assessment automations",
```

Comment:
Define x-ms-paging on collection API. Currently even if you don't need/support paging, by declaring the paging constructs, you can enable paging in future with out introducing a new api version.

Reply:
Comment applicable in other places too.